include/NeuraDialect/NeuraOps.td

```diff
@@ -89,8 +89,17 @@ def Neura_FDivOp : Op<NeuraDialect, "fdiv"> {
 // Defines a bitwise OR operation.
 def Neura_OrOp : Op<NeuraDialect, "or"> {
   let summary = "Bitwise OR operation";
-  let arguments = (ins AnySignlessInteger:$lhs, AnySignlessInteger:$rhs, Optional<AnyType>:$predicate);
-  let results = (outs AnySignlessInteger:$result);
+  let arguments = (ins AnyType:$lhs, AnyType:$rhs, Optional<AnyType>:$predicate);
```

Comment:
I mentioned in another PR, i.e., do you think we still need this `Optional<AnyType>:$predicate`? our data now has the predicate bit.

Reply:
Right, we may only need `grant_predicate` has a predicate as operand, which is not optional there. Please file an issue for this cleanup, thanks~!